src/coreComponents/fieldSpecification/FieldSpecificationBase.hpp

```diff
@@ -591,6 +599,8 @@ class FieldSpecificationBase : public dataRepository::Group
   /// The name of a function used to turn on and off the boundary condition.
   string m_bcApplicationFunctionName;
 
+  /// Value indicating whether we converts an error into a warning
+  string m_emptySetErrorMode;
```

Comment:
This is not how enum-strings are meant to be used.
```suggestion
  SetErrorMode m_emptySetErrorMode;
```